File: Coding with Mosh/Python/FizzBuzz.py
def fizz_buzz(input):
    if (input % 3 != 0 and input % 5 != 0):
        return input

    message = ""
    if (input % 3 == 0):
        message += "Fizz"

    if (input % 5 == 0):
        message += "Buzz"

    return message


print(fizz_buzz(15))
print(fizz_buzz(10))
print(fizz_buzz(6))
print(fizz_buzz(11))

File: Coding with Mosh/Java Script/Excercises/ControlFlow/fizzBuzz.js
console.log(fizzBuzz(3));
console.log(fizzBuzz(5));
console.log(fizzBuzz(15));
console.log(fizzBuzz(14));


function fizzBuzz(input) {
    if (typeof (input) !== 'number')
        return "Is not a number"

    let fizzy = "";

    if (input % 3 === 0)
        fizzy += "fizz";

    if (input % 5 === 0)
        fizzy += "buzz";

    return fizzy == "" ? input : fizzy;
}